accounts/utils.py

```diff
@@ -1,13 +0,0 @@
-from .models import User
-
-def create_cognito_accounts():
-    for user in User.objects.exclude(email=''):
-        if user.email == 'ginamlarson@gmail.com':
```

Comment:
Userdata in code=not cool Kyle

Reply:
That was committed by mistake. I'm removing it